frontend/src/components/Map/index.js

```diff
@@ -114,9 +119,46 @@ export default function Map({ draggable = true }) {
 
     map.addLayer(
       {
-        id: "maine",
+        id: "world-layer",
+        type: "fill",
+        layout: {},
+        maxzoom: countryMinZoom,
+        paint: {
+          "fill-color": {
+            type: "categorical",
+            property: "name",
+            stops: expression,
+          },
+          "fill-opacity": 0.7,
+          "fill-outline-color": fillOutlineColor,
+        },
+        source: {
+          type: "vector",
+          url: "mapbox://saurabhp.countries_tileset",
+        },
+        "source-layer": "countries",
+      },
+      "waterway-label"
+    );
+  };
+
+  const addNonUSLayer = async (map, data) => {
+    const covidData = await data;
+    const allexpression = covidData.map((row) => [
+      row.name,
+      getColor(row.group),
+    ]);
+    // Delete US from the world expression(all expression)
+    const expression = allexpression
+      .slice(0, 177)
+      .concat(allexpression.slice(178, allexpression.length));
```

Comment:
I think that matching the US data by `name` is safer than matching if by index - If in the future the api returns the countries in another order, or Us is no longer the 177th element this will not work as desired

What do you think of something like this?
```javascript
    const expression = covidData.filter(country => country.name !== "United States of America").map((row) => [row.name, getColor(row.group)]);

```

Reply:
Yes, you are right! Thanks for remind. I think your suggestion is good.